mcp/tools.go

```diff
@@ -972,3 +985,94 @@ func UniqueItems(unique bool) PropertyOption {
 		schema["uniqueItems"] = unique
 	}
 }
+
+// WithStringItems configures an array's items to be of type string.
+//
+// Supported options: Description(), DefaultString(), Enum(), MaxLength(), MinLength(), Pattern()
+// Note: Options like Required() are not valid for item schemas and will be ignored.
+//
+// Examples:
+//
+//	mcpgo.WithArray("tags", mcpgo.WithStringItems())
```

Comment:
please do not use alias package name `mcpgo`, replease it with `mcp`

Reply:
Hi @dugenkui03 
Thank you for the reminder.

The use of the alias mcpgo was due to a naming conflict in my local testing environment, where I have another package named mcp. I’ll update the import to use the correct package name mcp shortly. Apologies for the oversight, and thank you for pointing it out.